test/index.spec.js

```diff
@@ -2513,6 +2513,40 @@ describe('unexpected-dom', () => {
       `
       );
     });
+
+    describe('when used with expect.it', () => {
+      it('succeeds if the given structure is present', () => {
+        expect(
+          parseHtmlNode('<div></div>'),
+          'to satisfy',
+          expect.it('not to have attributes', 'class')
+        );
+      });
```

Comment:
Might be good to have a test of the output when it fails, too?

Reply:
Added.